xc7/tests/soc/litex/mini_ddr/arty.sdc

```diff
@@ -0,0 +1,13 @@
+create_clock -period 4.166 sys4x_clk__main_clkout_buf1 -waveform {0.000 2.083}
```

Comment:
Can we modify the input verilog to avoid net aliases and these strange net names for SDC constraints?

Reply:
Done.

The only clock that cannot be changed is the system clock. This because it gets renamed as it enters in submodules. I need to see how to overcome this issue.

I have also added the clk200 which wasn't there and is necessary to correctly place&route the FDPE flipflops for the reset signal.

Reply:
The solution may have to be fixing VPR.  Do have time to take a crack at that?

Reply:
Sure, I can take a look into that. We should be able to use aliases or sth like this for clock net names, to avoid this situation.